src/schema/v2/alerts.ts

```diff
@@ -0,0 +1,57 @@
+import {
+  GraphQLBoolean,
+  GraphQLInt,
+  GraphQLObjectType,
+  GraphQLString,
+} from "graphql"
+import { connectionWithCursorInfo } from "schema/v2/fields/pagination"
+import { ResolverContext } from "types/graphql"
+import { IDFields } from "./object_identification"
+import GraphQLJSON from "graphql-type-json"
+
+export const AlertType = new GraphQLObjectType<any, ResolverContext>({
+  name: "Alert",
+  fields: {
+    ...IDFields,
+    priceRange: {
```

Comment:
These are the fields/type that is a bit duplicitous to the stitched-in `SearchCriteria`, and in particular - it's the exact fields + types that is currently being consumed by Volt-V2.

Can tighten up the nullability.